src/components/Dashboard/index.jsx

```diff
@@ -60,10 +64,12 @@ import { user } from '../../utils/prop-types';
         position: 'fixed',
       },
       borderRight: 0,
+      backgroundColor:
+        theme.palette.type === 'dark' ? theme.palette.primary.main : '#fafafa',
```

Comment:
Wouldn't the light theme set its own `palette.primary.main`?

Reply:
You're absolutely right. I will send a fix :)